core/include/drivers/imx_caam_mkvb.h

```diff
@@ -0,0 +1,21 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright 2020 Pengutronix, Rouven Czerwinski <entwicklung@pengutronix.de>
```

Comment:
Minor: not the same email address as Author/S-o-b but you can leave it as is if you prefer.

Reply:
We prefer this for file headers, entwicklung (german for development) will reach all developers for questions even if the person may have left the company in the meantime.

Reply:
Das ist für mich kein Problem :wink: